pkg/asset/installconfig/vsphere/validation_test.go

```diff
@@ -80,185 +78,159 @@ func validMultiVCenterPlatform() *vsphere.Platform {
 	}
 }
 
-func TestValidate(t *testing.T) {
-	server := mock.StartSimulator()
-	defer server.Close()
+// simulatorHelper starts the govmomi simulator
+// returning the simulator.Server so that we can defer closing later and
+// shutdown the simulator to change versions.
+func simulatorHelper(t *testing.T, setVersionToSupported bool) (*validationContext, *simulator.Server, error) {
+	t.Helper()
+
+	server, err := mock.StartSimulator(setVersionToSupported)
+	if err != nil {
+		return nil, nil, err
+	}
 
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 
-	dcName := "DC0"
-	fName := "/F0"
-	dcName1 := "DC1"
-	tests := []struct {
-		name                      string
```

Comment:
do we want to strike the legacy tests while users could still be using the legacy spec?

Reply:
Once we hit installconfig/vsphere/validation.go we _should_ already converted the platform spec fields to `vcenters` and `failureDomains`

https://github.com/openshift/installer/pull/6770/commits/391df705068aa5101dd8538f1f85b96abc460ea5#diff-b626a7058aa0ea8b0c7b017e5ece4861c20c951f6eea20460721db83751f475dR19-R72

unless you see a problem there?

Reply:
yep, that makes sense.  i didn't encounter that logic until later in the review.